### PR TITLE
support for custom class on sezzle root and wrapper div added

### PR DESCRIPTION
**Pull request by sanbornsen**
- property name `customClasses`
- type : []
- format : `{xpath: '.', className: 'class-1'}`
- example - 
```
customClasses: [
    {xpath: '.', className: 'class-1'},
    {xpath: './div', className: 'class-2'}
]
```

### CODE REVIEW

**Comment by MartaJopp** [comment]
I think we should have it passed in as an array and map to targetXPath indices?

File: sezzle.js [hunk 1]
```diff
@@ -8,15 +8,11 @@ var SezzleJS = function (options) {
   if (options.targetXPath) {
     if (typeof (options.targetXPath) === 'string') {
       // Only one x-path is given
-      this.xpath.push(options.targetXPath.split('/').filter(function (subpath) {
-        return subpath !== '';
-      }));
+      this.xpath.push(this.breakXPath(options.targetXPath));
     } else {
       // options.targetXPath is an array of x-paths
       this.xpath = options.targetXPath.map(function (path) {
```
Comment (MartaJopp) [inline sezzle.js]: I think we have an issue with losing what `this` is. Could we set it to a variable? Testing was giving a type error that this.breakXPath is not a function.

File: sezzle.js [hunk 2]
```diff
@@ -604,6 +605,16 @@ SezzleJS.prototype.renderAwesomeSezzle = function (element, renderelement, index
...
+  this.customClasses.forEach(function(customClass) {
+    if (customClass.xpath && customClass.className) {
+      var path = this.breakXPath(customClass.xpath);
+      this.getElementsByXPath(path, 0, [sezzle])
+        .forEach(function(el) {
+          el.className += ' ' + customClass.className;
+        })
+    }
+  }.bind(this))
+
```
Comment (MartaJopp) [inline sezzle.js]: I really like the functionality to add custom classes to specific divs. Also, wondering if we can add custom class based on each widget. For example, if there are two widgets on the page and they only want a specific class on one of the widgets, or different class on each widget.

File: sezzle.js [hunk 3]
```diff
@@ -604,6 +605,16 @@ SezzleJS.prototype.renderAwesomeSezzle = function (element, renderelement, index
   // Adding main node to sezzel node
   sezzle.appendChild(node);
 
+  this.customClasses.forEach(function(customClass) {
```
Comment (MartaJopp) [inline sezzle.js]: I really like the functionality to add custom classes to specific divs. Also, wondering if we can add custom class based on each widget. For example, if there are two widgets on the page and they only want a specific class on one of the widgets, or different class on each widget.

File: sezzle.js (hunk 1 above)
Comment (sanbornsen) [reply sezzle.js]: Are you sure the error was coming from this line but not `610`?

File: sezzle.js (hunk 3 above)
Comment (sanbornsen) [reply sezzle.js]: Feature added

**Pull request merged**